src/main/app/claims/models/claim.ts

```diff
@@ -169,6 +170,17 @@ export class Claim {
   }
 
   get status (): ClaimStatus {
+    if (this.moneyReceivedOn) {
+      if (this.hasCCJ()) {
+        if (this.isCCJPaidWithinMonth()) {
```

Comment:
please use `&&`  operator instead if nested if block here

Reply:
I believe this needs to be nested, as there are 3 different claims statuses being returned depending on the result of each if statement.